### ISSUE TITLE
Sensitivity list 

### ISSUE THREAD

**Issue by rseac** [anikau31/systemc-clang#105]
# Description
The sensitivity list parsing does not capture function calls on classes that inherit from the sc_signal interface.  For example, `m_port.ready_chg()` in the zfp1 example.

**Comment by rseac** [comment]
This requires a rework of the `FindSensitivity` finder.  After analysis, it seems that the best approach would be the following.

1. Start by finding all the CXXCallOperatorExpr, and check it's arguments.
2. If the arguments are of certain types:
```
sc_sensitive& operator << ( const sc_event& );
sc_sensitive& operator << ( const sc_interface& );
sc_sensitive& operator << ( const sc_port_base& );
sc_sensitive& operator << ( sc_event_finder& );
```
then, we can further process the arguments passed as parameters.
3. The argument's instance pointer (FieldDecl, most likely) can be saved.

**Comment by rseac** [comment]
Fixed in becf12f
